ra2ce/analysis/indirect/multi_link_origin_destination.py

```diff
@@ -21,33 +22,27 @@
 
 
 class MultiLinkOriginDestination(AnalysisIndirectProtocol):
-    graph_file: GraphFile
+    analysis_input: AnalysisInputWrapper
```

Comment:
Is it really needed?

Reply:
It is passed to a next analysis. Made private.